x-pack/plugin/rollup/src/test/java/org/elasticsearch/xpack/rollup/job/RollupIndexerIndexingTests.java

```diff
@@ -705,10 +707,12 @@ private void executeTestCase(
         IndexSearcher searcher = new IndexSearcher(reader);
         String dateHistoField = config.getGroupConfig().getDateHistogram().getField();
         final ThreadPool threadPool = new TestThreadPool(getTestName());
+        final Client client = Mockito.mock(Client.class);
```

Comment:
Perhaps you would like to use `NoOpClient` here and everywhere else. Although Mockito should do its job fine here, we generally prefer this as to mocking the class directly.